ping.go

```diff
@@ -101,6 +107,26 @@ func NewPinger(addr string) (*Pinger, error) {
 	}, nil
 }
 
+// PingerConf represents options to configuring a pinger.
+type PingerConf struct {
+	// The address to ping.
```

Comment:
Please indicate in the comments which parameters are required and which are optional.